src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const changeProperties = {};
+
+  for (const key in robot) {
+    if (changeProperties.hasOwnProperty(key)
+    || changeProperties.hasOwnProperty(robot[key])) {
```

Comment:
you don;t need this check
```suggestion
    if (changeProperties.hasOwnProperty(robot[key])) {
```